buildozer/targets/android.py

```diff
@@ -1127,8 +1127,10 @@ def build_package(self):
             apk = u'{packagename}-{mode}.apk'.format(
                 packagename=packagename, mode=mode)
             apk_dir = join(dist_dir, "build", "outputs", "apk", mode_sign)
-            apk_dest = u'{packagename}-{version}-{mode}.apk'.format(
-                packagename=packagename, mode=mode, version=version)
+
+            apk_dest = u'{packagename}-{version}-{arch}-{mode}.apk'.format(
```

Comment:
We could move it outside the if/else and have that part made only once since we're doing exactly the same inside and outside the if/else